forge/routes/api/assistant.js

```diff
@@ -45,8 +45,10 @@ module.exports = async function (app) {
         }
     },
     async (request, reply) => {
-        // const method = request.params.method // FUTURE: allow for different methods
-        const method = 'function' // for now, only function node/code generation is supported
+        const method = request.params.method // the method to call at the assistant service
+        if (/^[a-z0-9_-]+$/.test(method) === false) {
```

Comment:
Should we test for the allowed functions here - ie `function`/`json`?

Reply:
I did consider that but I can (and do) evaluate them at the serice end.